scripts/deploy.sh

```diff
@@ -26,10 +26,17 @@ ls $DEPLOY_CONF_DIR
 cd $DEPLOY_SERVICE_DIR
 git config --global user.name "osscameroon-bot"
 
-#get repo url takes a `  Fetch URL: git@github.com:elhmn/infra.git`
-#and returns `@github.com/elhmn/infra.git`
-REPO_PATH=$(git remote show origin | grep Fetch | sed -E 's#^.*(git@|https://)([^/]+)/(.*)$#@\2:\3#;s#:#/#g')
-git pull --rebase https://$GITHUB_TOKEN$REPO_PATH main
+if [ ! -z "$GITHUB_TOKEN" ]; then
```

Comment:
```suggestion
if [ ! -z $GITHUB_TOKEN ]; then
```

you don't need double quotes, this token will never content spaces... right ? it's a single united string.

Reply:
I am not sure what you mean by united string, but the quotation are used because the shell might go completely crazy if not depending on what is the `$GITHUB_TOKEN` variable is set to. The shell expand this variable and if it has spaces and other weird characters the shell understands it is not going to work very well.